pypeit/par/pypeitpar.py

```diff
@@ -1029,16 +1029,19 @@ def __init__(self, ex_value=None, flux_value=None, nmaskedge=None,
         dtypes['nmaskedge'] = int
         descr['nmaskedge'] = 'Number of edge pixels to mask. This should be removed/fixed.'
 
-        # Offsets
         defaults['sn_smooth_npix'] = None
         dtypes['sn_smooth_npix'] = [int, float]
         descr['sn_smooth_npix'] = 'Number of pixels to median filter by when computing S/N used to decide how to scale ' \
                                   'and weight spectra. If set to None (default), the code will determine the effective ' \
                                   'number of good pixels per spectrum in the stack that is being co-added and use 10% of ' \
                                   'this neff.'
 
+        defaults['sigrej_exp'] = None
```

Comment:
For the default of None, you should not check rms_sn and should not mask in any way.

Reply:
See my response above.